registry/kusama.json

```diff
@@ -190,5 +190,13 @@
         "shortDescription": "CertHum.com - Community Validator",
         "longDescription": "CertHum is a community validator that provides Web3 infrastructure in support of decentralization.",
         "isOrganization": true
-     }
+     },
+    {
+        "address": "15vvAhHcmop8PjP5QNr3mXzX1oesv6vjun25iSgMe9YpDezv",
```

Comment:
For Kusama should be used address in Kusama format -https://wiki.polkadot.network/docs/learn-account-advanced#address-format
please, fix it and add proof of ownership